### PR TITLE
Add YSSY Curfew OPR INFO to ATIS section

### PR DESCRIPTION
**Pull request by AgentWolfFBI**
## Summary
Adds the curfew runway nomination OPR INFO line to the ATIS section for controllers who wish to simulate curfew mode operations.

## Changes
**Additions**:
- Curfew runway mode ATIS OPR INFO line.

### CODE REVIEW

File: docs/aerodromes/classc/Sydney.md [hunk 1]
```diff
@@ -296,6 +296,7 @@ The Operational Information field should be updated based on the runway mode and
 | `EXP INSTR APCH`, or<br>`EXP INSTR APCH THEN INDEP VISUAL APCH WHEN VISUAL` | `INDEP PARL DEPS IN PROG` |
 | `EXP INDEP VISUAL APCH` | `INDEP PARL APPROACHES AND DEPS IN PROG` |
 | SODROPS | `SIMUL OPP DIR PARL RWY OPS IN PROG` |
+| Curfew Mode Operations | `CURFEW RWY NOMINATION. CURFEW IN OPERATION UNTIL TIME 1900.` |
```
Comment (alphadelta332) [inline docs/aerodromes/classc/Sydney.md:299]: This is 2000 during standard time. Perhaps include a note about what this should be during DST vs ST?

**Review by alphadelta332 — CHANGES_REQUESTED** [review]
Not sure what that added temp.wav file is too, but I assume it doesn't need to be there

File: docs/aerodromes/classc/Sydney.md [hunk 2]
```diff
@@ -296,6 +296,8 @@ The Operational Information field should be updated based on the runway mode and
 | `EXP INSTR APCH`, or<br>`EXP INSTR APCH THEN INDEP VISUAL APCH WHEN VISUAL` | `INDEP PARL DEPS IN PROG` |
 | `EXP INDEP VISUAL APCH` | `INDEP PARL APPROACHES AND DEPS IN PROG` |
 | SODROPS | `SIMUL OPP DIR PARL RWY OPS IN PROG` |
+| Curfew Mode Operations | AEST: `CURFEW RWY NOMINATION. CURFEW IN OPERATION UNTIL TIME 2000.` |
+| | AEDT: `CURFEW RWY NOMINATION. CURFEW IN OPERATION UNTIL TIME 1900.` |
```
Comment (mattkelly4) [inline docs/aerodromes/classc/Sydney.md:300]:
```suggestion
| Curfew Mode Operations | AEST: `CURFEW RWY NOMINATION. CURFEW IN OPERATION UNTIL TIME 2000.` <br>AEDT: `CURFEW RWY NOMINATION. CURFEW IN OPERATION UNTIL TIME 1900.` |
```

Having these both in a single cell on new lines might help to correlate them with the title in column 1 better

**Review by alphadelta332 — APPROVED** [review]

**Review by mattkelly4 — APPROVED** [review]
Thanks for contributing this @AgentWolfFBI

**Review by tylerthetiletiler — APPROVED** [review]

**Pull request merged**